src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -863,7 +863,7 @@ private function loadDeleteForm(): void
         $this->template->assign('deleteForm', $deleteForm->createView());
     }
 
-    private function getHiddenJsonField(string $name, string $json): SpoonFormHidden
+    private function getHiddenJsonField(string $name, ?string $json = null): SpoonFormHidden
```

Comment:
The method always gets called with 2 param, the 2nd parameter is just `null` sometimes